docs/installation.md

```diff
@@ -1,9 +1,11 @@
-# Clone the repository
+# Installation and Configuration for cfgov-refresh
+
+## Clone the repository
 
 Using the console, navigate to the root directory in which your projects live and clone this project's repository:
 
 ```bash
-git clone git@github.com:cfpb/cfgov-refresh.git
+git clone https://github.com/cfpb/cfgov-refresh.git
```

Comment:
Agree with @jimmynotjim. My best guess is they recommend HTTPS because it's significantly easier for the (probably majority of people) that don't use 2FA, and marginally easier to set up the credential helper to cache your password/token than setting up an SSH key.